internal/widget/scroller.go

```diff
@@ -402,6 +402,9 @@ func (r *scrollContainerRenderer) handleShadowVisibility(offset, contentSize, sc
 }
 
 func (r *scrollContainerRenderer) updatePosition() {
+	if r.scroll.Content == nil {
```

Comment:
While trying to fix the bug about `window.Show`, I found this other bug, that causes a crash in fyne_demo with mobile tag, not sure why it worked before. But it seems to be unrelated to the PR #1986